build_debian.sh

```diff
@@ -318,6 +317,14 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     haveged                 \
     jq
 
+
+# Build ethtool package for generic platform
```

Comment:
Why build only for generic platform?

Reply:
on the Makefile it is configured to add this binary to SONIC_MAKE_FILES, for all other platforms it is ok and the binary is built.
For generic platform it is not, so in case we build it as generic we need to force it to make the binary.

Reply:
@lguohan: Does this seem like the proper approach here? Is there a cleaner method to get this installed in the generic image?

Reply:
A better solution was suggested by @nazariig (thanks!)
I applied it, can you please review it now?
@jleveque @lguohan